extensions/machine-learning-services/src/packageManagement/packageManagementService.ts

```diff
@@ -76,15 +77,41 @@ export class ServerConfigManager {
 	 * @param connection SQL Connection
 	 * @param enable if true external script will be enabled
 	 */
-	public async updateExternalScriptConfig(connection: azdata.connection.ConnectionProfile, enable: boolean): Promise<boolean> {
-		await this._queryRunner.updateExternalScriptConfig(connection, enable);
+	public async enableExternalScriptConfig(connection: azdata.connection.ConnectionProfile): Promise<boolean> {
```

Comment:
it seems we were try to incorporate enable/disable both but now only enable - why is that?

Reply:
yes, we only check if it's not enabled and give user option to enable it. Based on the recent design, enable/disable server config should not be here. we should add a dialog for all configs in admin extension maybe

---
In reply to: [393908973](https://github.com/microsoft/azuredatastudio/pull/9519#discussion_r393908973) [](ancestors = 393908973)

Reply:
why not disable? (for my understanding)

Reply:
this is prompted when user wants to do package management in database  which mean it needs the config to be enabled. there's not reason for user to disable it here. but user will have the option to disable any config in server config dialog which we have to add.

---
In reply to: [395374432](https://github.com/microsoft/azuredatastudio/pull/9519#discussion_r395374432) [](ancestors = 395374432)